server.py

```diff
@@ -69,6 +69,7 @@ def validate():
     })
 
     s = Schema({
+        Optional('heartbeat'): bool,
```

Comment:
Wondering if this needs to be a truthy value check as opposed to a boolean - everything else on the schema is a string. Deferring to @ajmaddaford.